setup.sh

```diff
@@ -10,6 +10,10 @@ echo "Creating virtual environment..."
 virtualenv -p $python venv
 source venv/bin/activate
 echo "Installing dependencies..."
-pip install -r requirements.txt
```

Comment:
Also, I would not write such a big commit message. Read [5 Useful Tips For A Better Commit Message](https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message)